orion/sites/admin/src/programs/program.js

```diff
@@ -21,35 +21,93 @@ export class ProgramPage extends React.Component {
         });
     }
 
+    onClickSelectAll() {
+        var items = document.getElementsByName("unpublished");
+        for (var i = 0; i < items.length; i++) {
+            if (items[i].type == "checkbox") {
+                items[i].checked = true;
+            }
+        }
+    }
+
+    onClickPublish() {
+        console.log("clicked publish");
+    }
+
     render() {
         const rows = this.state.list.map((row, index) => {
             return <ProgramRow key={index} row={row} />;
         });
         const numRows = rows.length;
+        let numUnpublished = 0;
+        let numSelected = 0;
         return (
             <div id="view-program">
-                <h1>All Programs ({numRows})</h1>
+                <ul>
+                    <h1>All Programs ({numRows}) </h1>
+                    <p>
+                        {" "}
+                        You have {numUnpublished} unpublished items. You have
+                        selected {numSelected} items to publish.{" "}
+                    </p>
+                </ul>
                 <ul id="list-heading">
+                    <button
+                        className="li-small"
+                        onClick={this.onClickSelectAll}>
+                        Select All
+                    </button>
                     <li className="li-med">ProgramKey</li>
                     <li className="li-med">Name</li>
                     <li className="li-small">Grade1</li>
                     <li className="li-small">Grade2</li>
                 </ul>
                 <ul id="list-rows">{rows}</ul>
-                <Link className="add-program" to={"/programs/add"}>
-                    Add Program
-                </Link>
+                <ul id="list-buttons">
```

Comment:
ok in this case, you should make a decision

This is either a "list" in which case you use a `<ul>` or it's a "container" (i.e. buttons-container) in which case it's just a regular `<div>`. If you go with the list, the insides should be `<li>`s. Otherwise divs inside divs are fine. In this case, I would do the divs.